Programs/python.c

```diff
@@ -15,6 +15,110 @@ wmain(int argc, wchar_t **argv)
 }
 #else
 
+/* Helpers to better handle the legacy C locale
```

Comment:
There's been some discussion about the "legacy C locale" nomenclature.  What about "bare C locale"?